src/Nethermind/Nethermind.Crypto/IEcdsa.cs

```diff
@@ -8,7 +8,10 @@ namespace Nethermind.Crypto
     public interface IEcdsa
     {
         Signature Sign(PrivateKey privateKey, Keccak message);
+        Signature Sign(PrivateKey privateKey, in ValueKeccak message);
```

Comment:
if we can get ValueKeccak out of Keccak, maybe we should remove Keccak overloads in all the classes/interfaces? Maybe add implicit cast to ValueKeccak?